modules/rest-util/test/blaze/middleware/fhir/output_test.clj

```diff
@@ -136,9 +136,11 @@
       [:headers "Content-Type"] := nil
       :body := nil))
 
-  (testing "failing XML emit"
+  (testing "invalid XML"
```

Comment:
"failing XML emit" was better, because there is no invalid XML, there is a Patient in the internal FHIR representation that can't be emitted as XML, but it's no XML in the first place. so the function from internal FHIR representation to XML is only partial.

Reply:
I see.
Then, what about "failed conversion (XML)", or similar?

Reply:
Can you please keep the doc. It was already good.